New features

sti-233 opened the pull request:
- 新增 屏蔽视频热搜横幅
- 新增 强制启用分区筛选
- 新增 添加旧版分区入口
- 新增 隐藏直播分区Banner卡片
   └ Resolve #611 
- 修复 首页推荐选择本地漫游屏蔽后不生效
   ├ Resolve #672
   └ Resolve #720
- 修复 恢复旧版评论区样式失效
   └ Resolve #686
- 更新 grpc_apis.jar

sti-233 committed:
feat: hide live banner

sti-233 commented:
视频热搜横幅部分为下图
"热搜第14名·如何回到五一放假前"
![1000270219](https://github.com/user-attachments/assets/ad9663f2-752c-4e96-b029-4da0225e8828)

关于分区筛选：
左 被官方停用  右 强制启用 实测有效
![1000270229](https://github.com/user-attachments/assets/6e6e5c9b-b522-4ce0-8bdd-08002f4977b7)

关于旧版分区入口：
`小伙伴由于产品前端样式调整，二级分区相关入口将从2025年2月27日起逐步灰度下线，在网页版、App（V8.35及以上版本）生效，您可以通过搜索功能查找感兴趣的内容~`
下图为旧版本分区页面的二级入口
![114514](https://github.com/user-attachments/assets/ac144bb8-34e3-408b-8129-628df9d8be8d)

开启后将添加至新版入口上方
![Screenshot_2025-05-05-17-23-43-839_tv danmaku bili](https://github.com/user-attachments/assets/0563284e-b00b-404f-b222-f383fb241bfd)

zjns commented:
不要用 XXXReply.toBuilder 或者 XXXReplyBuilder.build() 修改 protobuf 实体，而是直接 XXXReply.set/add/remove/clear 修改，XXXReplyBuilder 会被混淆。

sti-233 commented:
> 不要用 XXXReply.toBuilder() 或者 XXXReplyBuilder.build() 修改 protobuf 实体，而是直接 XXXReply.set/add/remove/clear 修改，XXXReplyBuilder 会被混淆。grpc_apis.jar 也应该是在 patch 后的 apk 中提取出来。

好的，到时候我改改

sti-233 commented:
> 不要用 XXXReply.toBuilder() 或者 XXXReplyBuilder.build() 修改 protobuf 实体，而是直接 XXXReply.set/add/remove/clear 修改，XXXReplyBuilder 会被混淆。grpc_apis.jar 也应该是在 patch 后的 apk 中提取出来。

好的，我到时候改一下

sti-233 commented:
应该是改好了()

zjns commented:
newBuilder什么的也不要用，直接实例化

sti-233 commented:
> newBuilder什么的也不要用，直接实例化

改好了

zjns commented:
RegionList 里的那几个 val 属性有必要定义在类上吗？object 类是会静态单例一直存在于内存中，非必要应该挪到方法内去。

sti-233 committed:
feat: about partition & block

feat: force enable partition filter
feat: add old region
feat: block hot search
chore: update grpc_apis.jar

sti-233 commented:
> RegionList 里的那几个 val 属性有必要定义在类上吗？object 类是会静态单例一直存在于内存中，非必要应该挪到方法内去。

修改好了()

sti-233 committed:
fix: patch onFeedClick

sti-233 committed:
fix: force_old_comments

sti-233 committed:
fix: patch onFeedClick

z0z0r4 commented:
什么时候会合并呢？有点急需，关键词屏蔽炸了
